cmd/weaviate-core/main.go

```diff
@@ -0,0 +1,145 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2023 Weaviate B.V. All rights reserved.
+//
+//  CONTACT: hello@weaviate.io
+//
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/weaviate/weaviate/adapters/handlers/rest"
+	"github.com/weaviate/weaviate/entities/storobj"
+	"github.com/weaviate/weaviate/usecases/config"
+)
+
+func main() {
+	dataPath := flag.String("data-path", "./data", "path to the data directory")
+	origin := flag.String("origin", "http://localhost:8080", "listen address")
+	clusterHostname := flag.String("cluster-hostname", "node1", "cluster hostname")
+	defaultVectoriser := flag.String("default-vectoriser", "none", "default vectoriser")
```

Comment:
`vectoriser` is spelled `vectorizer` elsewhere in the codebase

Reply:
I don't accept your fancy new language innovations :P